docs/src/test/bash/build-helper.sh

```diff
@@ -21,12 +21,17 @@ if [[ $# -ne 1 ]]; then
 fi
 
 SHELLCHECK_VERSION="v0.6.0"
+ARCH="x86_64"
+if [ "$(uname -m)" == aarch64 ]; then
+    SHELLCHECK_VERSION="v0.7.0"
```

Comment:
Let's go ahead and update the version regardless of arch

Reply:
Updated the version as v0.7.0 for both arch